OWNERS_ALIASES

```diff
@@ -11,3 +11,43 @@ aliases:
   - jottofar
   - vrutkovs
   - wking
+  team-leads:
+  - JAORMX # Infrastructure Security and Compliance
+  - JoelSpeed # Cluster Infrastructure
+  - LalatenduMohanty # Over the Air Updates
+  - Miciah # Network Edge
+  - adambkaplan # Build + Jenkins
+  - ashishkamra # Performance and Latency Sensitive Application Platform
+  - beekhof # Machine Lifecycle
+  - brenton # Technical Release Team
+  - bthurber # Partner Accelerators
+  - celebdor # Ecosystem Edge
+  - dmage # Image Registry
+  - hexfusion # etcd
+  - jaypoulz # Multi Arch
+  - jcantrill # Cluster Logging
+  - jmrodri # Operator SDK
+  - joelddiaz # Hive
+  - jparrill # Portfolio Integration
+  - jsafrane # Storage
+  - jwmatthews # App Migration
+  - kevinrizza # Operator Lifecycle Manager
+  - mandre # Shift on Stack
+  - miabbott # CoreOS
+  - dulek # Kuryr Networking
+  - pedjak # App Services
+  - romfreiman # Single Node OpenShift
+  - rphillips # Node
+  - s-urbaniak # Authentication
+  - sadasu # Metal Platform
+  - selansen # Windows Containers
+  - simonpasquier # Monitoring - In Cluster
+  - sinnykumari # MCO
+  - soltysh # Workloads
+  - sosiouxme # Automated Release Tooling
+  - staebler # OpenShift Installer
+  - stevekuznetsov # Test Platform
+  - sttts # API Server
+  - thomasmckay # Quay-Quay.io
```

Comment:
He is no longer in the Quay team. But I'm almost sure that the Quay team doesn't use this repo.

Reply:
If these team leads are not accurate, folks should update the Shiftzilla config in the li repository.  But I've pushed 81c439a4bc -> fc34a9be48 to drop Quay entirely, so there's no need to hold this PR on an li fix for this entry.